frontend/public/components/about-modal.tsx

```diff
@@ -162,16 +162,18 @@ AboutModalItems.displayName = 'AboutModalItems';
 export const AboutModal: React.FC<AboutModalProps> = (props) => {
   const { isOpen, closeAboutModal } = props;
   const { t } = useTranslation();
-  const details = getBrandingDetails();
-  const customBranding = window.SERVER_FLAGS.customLogoURL || window.SERVER_FLAGS.customProductName;
+  const { productName } = getBrandingDetails();
+  const customLogoUrl = useCustomLogoURL(MASTHEAD_TYPE);
+
+  const customBranding = customLogoUrl || window.SERVER_FLAGS.customProductName;
   const openShiftBranding = window.SERVER_FLAGS.branding !== 'okd' && !customBranding;
   return (
     <PfAboutModal
       isOpen={isOpen}
       onClose={closeAboutModal}
-      productName={details.productName}
-      brandImageSrc={openShiftBranding && redHatFedoraImg}
-      brandImageAlt={openShiftBranding && details.productName}
+      productName={productName}
+      brandImageSrc={openShiftBranding ? redHatFedoraImg : customLogoUrl}
+      brandImageAlt={openShiftBranding && productName}
```

Comment:
Not a new bug, but please make the following change since you're already changing this code.
```suggestion
      brandImageAlt={productName}
```

Reply:
I have adjusted this logic a bit in the newest commit into 
```  
      brandImageSrc={customLogoUrl || (openShiftBranding && redHatFedoraImg)}
      brandImageAlt={(openShiftBranding || customLogoUrl) && productName}
```
Where I've set the prioritised logo source to be coming from `customLogoUrl`, the fallback is `redHatFedoraImg` depending on the truthiness of `openShiftBranding` - we don't want to show any logos for non-openshift brandings (okd).
The problem with prior logic was that whenever the `openShiftBranding`, we have always used `customLogoUrl` for the `brandImageSrc` (even when customLogos are not configured). Which resulted into an empty icon being shown in the modal. 
The other problem is tied to your suggestion, setting the `brandImageAlt` to always be `productName` created a new problematic edge case - the `openShiftBranding` being False (console branding is set to okd) and there are no custom logos configured. This resulted into an empty icon being shown again because we have set the `brandImageAlt` but `brandImageSrc` remained empty.
![Screenshot 2025-04-15 at 14 05 37](https://github.com/user-attachments/assets/db9a5e84-b30f-43fe-9bc2-f4b1f304fa1d)